vault/example_keys.json

```diff
@@ -2,4 +2,4 @@
   "new_relic_key": "To do"
 }
 
-// Change file name to keys.json. Remove this line
\ No newline at end of file
+// Change file name to keys.json and remove this line
```

Comment:
Can you clarify why this was changed and why it's still here?

Reply:
This is here so the next person who receives the repository knows what to change, and how to make the keys work. In a way it's a .env type file. I set it this way so it's easier for me to open it up on an EC2 instance and modify the credentials.  Whoever receives this repository next will know exactly how to login rather than second guessing their .env file.